apps/bartender.js

```diff
@@ -51,18 +58,43 @@ $(document).ready(function() {
     $('.answer-buttons').show();
   });
 
-  // Answer to drink preferences
+  // Answer to drink preferences, choices added to preferences array
 
   var questionNumber = 0;
   $('#yes, #no').click(function(e) {
     e.preventDefault();
     if (this.id === 'yes') {
-      customerPreferences[questionNumber] = true;
-    } else if (this.id === "no") {
-      customerPreferences[questionNumber] = false;
+      customerPreferences.preferences[questionNumber] = true;
+      questionNumber++;
+      $('.bar-questions').text(barQuestions.questions[questionNumber]);
+    } else if (this.id === 'no') {
+      customerPreferences.preferences[questionNumber] = false;
+      questionNumber++;
+      $('.bar-questions').text(barQuestions.questions[questionNumber]);
+    }
+
+    // All questions answered
+
+    if (questionNumber === 5) {
+      $('.answer-buttons').hide();
+      $('.create-drink').show();
+      $('.bar-questions').text("Have the Bartender Mix Your Drink!");
+    }
+  });
+
+  // Bartender mixes drink and displays ingredients used to customer
+
+  $('#create').click(function(e) {
+    e.preventDefault();
+    var pirateBartender = new Bartender();
+    var drink = pirateBartender.createDrink(customerPreferences.preferences);
+    var last = drink.length - 1;
+    $('.bar-questions').text("Here's your drink mixed with");
+    for (var i = 0; i < drink.length - 1; i++) {
+      $('.bar-questions').append(", a " + drink[i].toString());
```

Comment:
I think `drink[i]` is already a string so the `.toString()` might not be doing anything. I might be wrong.